src/TBuffer.cpp

```diff
@@ -3525,7 +3525,7 @@ bool TBuffer::processUtf8Sequence(const std::string& bufferData, const bool isFr
             for (size_t i = 0; i < utf8SequenceLength; ++i) {
                 debugMsg.append(QStringLiteral("<%1>").arg(static_cast<quint8>(bufferData.at(pos + i)), 2, 16, QChar('0')));
             }
-            qDebug().nospace() << "    Sequence bytes are: " << debugMsg.toLatin1().constData();
+            qDebug().nospace() << "    Sequence bytes are: " << debugMsg;
```

Comment:
🤔 As the string that is used here (and the next two proposed changes) is strictly ASCII in this case I guess this is okay.